data/hai/hai reveal 4 middle.txt

```diff
@@ -1456,32 +1458,410 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	Some of the details in the report were secrets the Hai shouldn't have known about know about. You recognize some of the information as private war planning discussions known only to yourself and a handful of other people. While there are obvious knowledge gaps, the level of detail they knew of the command hierarchy, fleet movements, special ops missions, intelligence services, and weapons designs, is quite disturbing.`
+			`	This points to a vast intelligence network; it seems Hai aren't as naive as their cute furry exterior implies. You look up at the functionary, and she gives you a friendly little smile that seems more sinister now than it did a minute ago.`
+			`	Of course, if their intelligence agencies are so capable, shouldn't they have been able to solve their pirate problems?`
+				goto reading
+				to display
+					not "Hai Reveal [A11] Sound the Alarm: pirates book"
+			`	It seems that, although the Hai have trouble tracking small, independent, anarchist, groups, they have no trouble with large, established, hierarchies. Their very approach to anarchists was flawed: make deals with the biggest groups, as if those were heads of an anarchist hierarchy. The weakness of the Hai intelligence service is asymmetric warfare.`
+				goto reading
+				to display
+					has "Hai Reveal [A11] Sound the Alarm: pirates book"
+
+			label pirates
+			action
+				log `The Hai intelligence agencies' approach to pirates was to befriend the large anarchist groups, as if they were the head of an "anarchist hierarchy. This suggests they lack understanding of raw chaos; in essence, their greatest weakness is asymmetric warfare.`
+				set "Hai Reveal [A11] Sound the Alarm: pirates book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+			`	The report begins with a brief summary of Hai involvement with anarchist groups, which dates back from the founding of the first anarchist colony, Freedom, around the same time as human first contact. Apparently, the Hai have had generally positive relations with the largest organizations on anarchist planets. The Hai trade humanitarian supplies or other goods for assistance from the anarchist planets, even ones as distant as Thule and Greenrock.`
+			`	Much of the nature of the "assistance from anarchist planets" is redacted, but it seems it mostly involves shutting down groups that try to reveal or harm the Hai. However, there are notes about deals with anarchists who oppose slavery, arranging massive rescues, bringing thousands of slaves at a time to freedom in Hai space. Also, there is an amnesty program for pirates who have committed no serious crimes, which explains the abundance of reformed pirates on Allhome.`
```

Comment:
I'd love if this was presented another way: they found the old pirates to help their own, given they know the place and stuff and wont attract any attention

Reply:
Hai Reveal already mentions a pattern of funding large anarchist organizations. I'm merely elaborating on that point.

Reply:
Yeah and I think it'd be smarter to do it by founding old pirates

Reply:
The intelligence reports do not describe how the Hai achieve what they do. This is low-level intelligence, which only provides results, not sources.